util/processinfo.go

```diff
@@ -95,3 +95,15 @@ type SessionManager interface {
 	GetProcessInfo(id uint64) (*ProcessInfo, bool)
 	Kill(connectionID uint64, query bool)
 }
+
+var globalSessionManager SessionManager
```

Comment:
Do we really need this global variable?

Reply:
After consideration, we can remove it now.